bitnami/charts
# [bitnami/fluentd] Fix default value for readOnlyRootFilesystem

**Description of the change**

Since the removal of `nil` values as defaults in the _values.yaml_ (see #7024), the installation of this Helm Chart with specific parameters is not working with the following error:

```console
$ helm install mychart --set forwarder.containerSecurityContext.capabilities.drop=null --set aggregator.containerSecurityContext.capabilities.drop=null .
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(DaemonSet.spec.template.spec.containers[0].securityContext.readOnlyRootFilesystem): invalid type for io.k8s.api.core.v1.SecurityContext.readOnlyRootFilesystem: got "string", expected "boolean"
```

This PR set a default boolean value for the `readOnlyRootFilesystem` parameter, with those changes:
```console
$ helm install mychart --set forwarder.containerSecurityContext.capabilities.drop=null --set aggregator.containerSecurityContext.capabilities.drop=null .
NAME: mychart
LAST DEPLOYED: Thu Jul 29 08:56:48 2021
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
** Please be patient while the chart is being deployed **

  To verify that Fluentd has started, run:

    kubectl get all -l "app.kubernetes.io/name=fluentd,app.kubernetes.io/instance=mychart"

  Logs are captured on each node by the forwarder pods and then sent to the aggregator pods. By default, the aggregator pods send the logs to the standard output.
  You can see all the logs by running this command:

    kubectl logs -l "app.kubernetes.io/component=aggregator"

  You can mount your own configuration files to the aggregators and the forwarders. For example, this is useful if you want to forward the aggregated logs to Elasticsearch or another service.
```
